compute/resource_cluster.go

```diff
@@ -82,17 +86,39 @@ func resourceClusterSchema() map[string]*schema.Schema {
 			Type:     schema.TypeMap,
 			Computed: true,
 		}
+		s["num_workers"] = &schema.Schema{
+			Type:             schema.TypeInt,
+			Optional:         true,
+			Default:          0,
+			ValidateDiagFunc: validation.ToDiagFunc(validation.IntAtLeast(0)),
```

Comment:
either add `// nolint` or replace with new validation framework, otherwise `make lint` fails

Reply:
it was originally failing, but was fixed by bumpting to newer SDK:

```
>make fmt && make lint                13:47 0
✓ Formatting source code with goimports ...
✓ Formatting source code with gofmt ...
✓ Filling vendor folder with library code ...
✓ Linting source code with golangci-lint make sure you run make fmt ...
```